cartography/data/jobs/cleanup/azure_sql_server_cleanup.json

```diff
@@ -0,0 +1,125 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:AzureSQLServer)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (:AzureSQLServer)<-[r:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (n:AzureServerDNSAlias)<-[:USED_BY]-(:AzureSQLServer)<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
Any reason to include this in the queries? `<-[:RESOURCE]-(:AzureSubscription{id: {AZURE_SUBSCRIPTION_ID}})`
I think the queries should match just fine without the extra constraint.

Reply:
@ramonpetgrave64 I used the cleanup of the AWS codes and other codes as reference over here and they follow the same approach of specifying the entire resource hierarchy. Also, I used the same approach for the AWS KMS and API Gateway modules we added recently. So, for consistency, I used the same approach in the Azure cleanup codes. Didn't wanna be the cause of any weird errors.

Reply:
Yeah, keep the whole resource hierarchy - we don't want to have a sync delete resources from a subscription that is not the current subscription.